src/arith_uint256.cpp

```diff
@@ -224,6 +230,16 @@ template void base_uint<256>::SetHex(const std::string&);
 template unsigned int base_uint<256>::bits() const;
 template base_uint<256> base_uint<256>::sqrt() const;
 
+template base_uint<128>& base_uint<128>::operator<<=(unsigned int);
+template base_uint<128>& base_uint<128>::operator>>=(unsigned int);
+template base_uint<128>& base_uint<128>::operator*=(uint32_t b32);
+template base_uint<128>& base_uint<128>::operator*=(const base_uint<128>& b);
+template base_uint<128>& base_uint<128>::operator/=(const base_uint<128>& b);
+template int base_uint<128>::CompareTo(const base_uint<128>&) const;
+template bool base_uint<128>::EqualTo(uint64_t) const;
+template std::string base_uint<128>::GetHex() const;
+template std::string base_uint<128>::ToString() const;
```

Comment:
It's redundant, the above as well. Explicit template specialization can be based on entire class not on specific function.
`template<> class base_uint<128>;`